src/main/java/preponderous/ponder/minecraft/bukkit/tools/MaterialChecker.java

```diff
@@ -0,0 +1,68 @@
+package preponderous.ponder.minecraft.bukkit.tools;
+
+import org.bukkit.Material;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * @author Daniel McCoy Stephenson
```

Comment:
If this is a new class that you've contributed, you don't need to put me as an author :)